fda/basis.py

```diff
@@ -253,6 +284,24 @@ def penalty(self, derivative_degree=None, coefficients=None):
         """
         pass
 
+    def rescale(self, domain_range=None):
+        r"""Return a copy of the basis with a new domain range, with the
+            corresponding values rescaled to the new bounds.
+
+            Args:
+                domain_range (tuple, optional): Definition of the interval where
+                    the basis defines a space. Defaults uses the same as the
+                    original basis.
+        """
+
+        if domain_range == None:
+            domain_range = self.domain_range
+        elif domain_range[0] >= domain_range[1]:
+            raise ValueError("The interval {} is not well-defined.".format(
```

Comment:
As we are targeting only Python version 3.6 or above, I think we should use [f-strings](https://www.python.org/dev/peps/pep-0498/) as they are more legible than format.

Reply:
At the moment the repository is compatible with python 3.5, adding these changes would make it incompatible and travis would not compile correctly. 
https://github.com/GAA-UAM/fda/blob/c09ec2253f10d60157b4ceb453d3e19d53eed4a1/.travis.yml#L2-L3
I think it is not a good idea to add these changes until we decide which version to use.